Sources/InstantSearchCore/Telemetry/Telemetry.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  Telemetry.swift
+//  
+//
+//  Created by Vladislav Fitc on 06/10/2021.
+//
+
+import Foundation
+import InstantSearchTelemetry
+
+typealias Telemetry = InstantSearchTelemetry
+
+extension Telemetry: UserAgentExtending {
+
+  public var userAgentExtension: String {
+    return encodedValue.flatMap { "InstantSearchTelemetry(\($0))" } ?? ""
```

Comment:
_Suggestion_: maybe we can simply make it `Telemetry(<data>)` instead of `InstantSearchTelemetry(<data>)` ?

Reply:
As it will be a part of raw user-agent string and will be parsed with regex, I prefer to keep it clearly related to InstantSearch. We never now if there is another library sending the telemetry data in a similar way. To make it concise, we can use prefix, e.g. `ISTelemetry(<data>)`. WDYT?

Reply:
Yes,`ISTelemetry(<data>)` should be better imo.

Reply:
```suggestion
    return encodedValue.flatMap { "ISTelemetry(\($0))" } ?? ""
```